### DISCUSSION TITLE
stenc-2.0.0_rc

### DISCUSSION THREAD

**Discussion by jonasstein** [scsitape/stenc#110] [Announcements]
release candidate

<hr /><em>This discussion was created from the release <a href='https://github.com/scsitape/stenc/releases/tag/stenc-2.0.0_rc2'>stenc-2.0.0_rc2</a>.</em>

Beta testers, please test and report about the release candidate for 2.0.0. The old source could not compile with gcc-13, this should be fixed now.

@sunwire, Fedora, amd64, HP StorageWorks Ultrium 1760 LTO-4 SAS
@jmwilson, Debian + FreeBSD, amd64, HP LTO-6
@DaveQB , FreeBSD, AMD64, IBM ULT3580-HH5

**Comment by sunwire** [comment]
@jonasstein It compiles without a problem on Fedora 37,38 and rawhide (gcc 13.1) and CentOS Stream + EPEL 8 and 9 tested on real hardware on Fedora 37. Everything works well.

**Comment by jmwilson** [comment]
Exercised all options and modes on AMD64 FreeBSD 13.2 + HP LTO 6, no issues.
